core/src/credential.rs

```diff
@@ -0,0 +1,147 @@
+
+#![allow(unused_variables)]
+#![allow(dead_code)]
+use std::{error::Error, time::{Instant, SystemTime}};
+
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+// cred_subject is a generic that implements trait X
+// trait X allows us to encode that object into JSON-LD
+// We provide types that implement trait X for the cred types that we support
+// Users can also user their own types that implement trait X if they need a different structure
+// --- 
+// Default context and Cred types are defaulted but can be redefined 
+
+type VCContext = [&'static str];
```

Comment:
Why make this a custom type? And also I think we should rename this type (if we're gonna keep it) into something more generic (since its for both VCs and VPs)

Reply:
The custom type thing has been addressed, though the rename would still be valuable